clinica/pipelines/statistics_volume/statistics_volume_utils.py

```diff
@@ -238,72 +365,107 @@ def write_covariate_lines(m_file_to_write_in, covar_number, covar_name, covar_va
     m_file.close()
 
 
-def run_m_script(m_file):
-    """
-        Runs a matlab m file for SPM, determining automatically if it must be launched with SPM or SPM Standalone
-        If launch with spm standalone, the line 'spm_jobman('run', matlabbatch)' must be removed because unnecessary
+def run_m_script(m_file: str) -> str:
+    """Runs a Matlab file for SPM.
 
-    Args:
-        m_file: (str) path to Matlab m file
+    Determines automatically if the script should be launched with SPM or SPM standalone.
+    If launch with spm standalone, the line 'spm_jobman('run', matlabbatch)' must be removed because unnecessary
 
-    Returns:
-        output_mat_file: (str) path to the SPM.mat file needed in SPM analysis
+    Parameters
+    ----------
+    m_file: str
+        Path to Matlab m file
+
+    Returns
+    -------
+    output_mat_file: str
+        Path to the SPM.mat file needed in SPM analysis
     """
-    import platform
-    from os import system
-    from os.path import abspath, basename, dirname, isfile, join
 
-    from nipype.interfaces.matlab import MatlabCommand, get_matlab_command
+    from os.path import abspath, dirname, isfile, join
+    from pathlib import Path
```

Comment:
For the same reason:

```suggestion
    from pathlib import Path
    from clinica.pipelines.statistics_volume.statistics_volume_utils import (  # noqa
        run_matlab_script_with_spm_standalone,
        run_matlab_script_with_matlab,
    )
```